cranelift-codegen/src/regalloc/liverange.rs

```diff
@@ -167,29 +179,18 @@ pub struct GenericLiveRange<PO: ProgramOrder> {
 
     /// Additional live-in intervals sorted in program order.
     ///
-    /// This map is empty for most values which are only used in one EBB.
+    /// This vector is empty for most values which are only used in one EBB.
     ///
-    /// A map entry `ebb -> inst` means that the live range is live-in to `ebb`, continuing up to
+    /// An entry `ebb -> inst` means that the live range is live-in to `ebb`, continuing up to
```

Comment:
```suggestion
    /// An entry `ebb -> inst` means that the live range is live at the start of `ebb`, continuing up to
```

Reply:
Actually, live-in is the term of art and is used elsewhere too, so I sort of think it would be good to revert this change.